tests/e2e/backup_restore_suite_test.go

```diff
@@ -79,13 +78,9 @@ var _ = Describe("AWS backup restore tests", func() {
 			}
 
 			if brCase.BackupRestoreType == csi {
-				if clusterProfile == "aws" {
-					log.Printf("Creating VolumeSnapshot for CSI backuprestore of %s", brCase.Name)
-					err = installApplication(dpaCR.Client, "./sample-applications/gp2-csi/volumeSnapshotClass.yaml")
-					Expect(err).ToNot(HaveOccurred())
-				} else {
```

Comment:
@mperetzred could you please explain, reason for removing this else?

Reply:
no need for it, using aws label instead, and it will be filtered if cluster profile is not aws (check the filters on the makefile)